server/server.js

```diff
@@ -1043,6 +1043,8 @@ let needSetup = false;
             try {
                 checkLogin(socket);
 
+                log.debug("manage", `Add Tag: ${tag} User ID: ${socket.userID}`);
```

Comment:
Are the changes in this file intentional or a rebasing artefact?

Reply:
Intentional kind of.... used for testing the functionality. But leaved them in Intentional as they are only logged to debug. But if they are unwanted... I can remove them.